topic: "inertialsense"
port: [/dev/ttyACM0, /dev/ttyACM1]
baudrate: 921600
publishTf: false                                 # Publish Transform Frame (TR)
frame_id: ""                                    # FIXME: What is this?  is it just the FrameID to use in the ROS messages?
ins:
  enable_covariance: true                       # Include covariance data in odom_ins_... messages
  messages:
    odom_ins_enu:
      topic: "odom_ins_enu"
      enable: true
    did_ins4:
      enable: false  # we dont need the publisher, only the messages
      period: 50     # if odom_ins_* enabled pimu period will be used

gps1:
  type: 'F9P'

rtk_rover:
  positioning_enable: true                     # Enable RTK precision positioning at GPS1
  compassing_enable: true                      # Enable RTK compassing (dual GNSS moving baseline RTK) at GPS2
  correction_input:
    select: radio                       # Select correction input settings

    radio:
      type: "evb"
      format: RTCM3
      port: "LoRa"
